audformat/core/database.py

```diff
@@ -101,9 +109,16 @@ def __init__(
             expires: datetime.date = None,
             languages: typing.Union[str, typing.Sequence[str]] = None,
             description: str = None,
+            license: typing.Union[str, define.License] = None,
```

Comment:
`define.License` as a type cannot be correct as it is no enumeration. But we use the same with `usage` for example.

Reply:
I'm fine with it, unless you would like to change to `enum.Enum` as we do in `opensmile.FeatureExtractor`. But there was some reason I didn't like to work `enum.Enum`, though I cannot remember what it was at the moment. The other problem might be that we also derive from `DefineBase` here, so not sure if it is straightforward to change to `enum.Enum`.

Reply:
No, I don't want to change to `enum.Enum`, but I think we can only write `license: str = None` and `usage: str = define.Usage.UNRESTRICTED` and have to refer to `define.License` and `define.Usage` in the docstrings.

Reply:
Because `type(audformat.define.License.CC0_1_0) != audformat.define.License`

Reply:
Ok, maybe that makes more sense. But we actually have it in quite a lot of places, so I suggest to do it in a separate MR and leave it as it is for the moment.

Reply:
OK, I opened https://github.com/audeering/audformat/issues/39